servicepulse/redirect.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Redirects
+summary: Describes what message redirects are and how to use them.
+tags:
+- ServicePulse
+related:
+- servicepulse/troubleshooting
+---
+
+## Message Redirects
+
+In Versions **1.6.6** and above ServicePulse includes a screen to view and manage failed message redirects that have been created to send failed messages to an alternate queue when the original processing endpoint address is no longer available.
+
+INFO: Message redirects are only a feature of ServicePulse/ServiceControl and will not alter the routing for NServiceBus endpoints.
+
+To manage redirects go to Configuration page and select tab called `Retry Redirects`.
+
+![Redirects Tab](images/redirects.png 'width=500')
+
+### Managing redirects
+On the redirect page a list of redirects is being displayed containing following information.
+ - **Source Queue** This is the queue for which this redirect will be applied.
+ - **Destination Queue** This is the queue that will be new destination when doing retry.
+ - **Last modified** This is a timestamp of last modification.
+
+To create a redirect click the `Create Redirect` button. A dialog will appear.
+
+![Create Redirects Dialog](images/redirects-create.png 'width=500')
+
+Choose a source queue from the dropdown. Enter the queue name for the target of the redirect in `To Physical Address` input. To immediately retry all unresolved failed messages from the sources address check the box with the label `Immediately retry any matching failed messages`.
+
+Once created every redirect can be deleted using `End Redirect` link. This action requires an confirmation. 
+
+To change the target of a redirect click the `Modify Redirect` link. Only the target queue can be modified, the source queue remains read-only.
+
+#### Validation errors
+
+When adding and editing redirects following validation rules are checked:
+ - **Duplicate** There can not be more than one rule for given Source Queue. In that case the following message will be displayed: "Can not create redirect to a queue *QueueName* as it already has a redirect. Provide a different queue or end the redirect."
```

Comment:
As the message can be confusing by keeping it mostly the same I allow the user to google our doco page, which sheds more light. I vote for leaving the error message